docs/asyncapi.yaml

```diff
@@ -115,6 +119,212 @@ components:
                 accountCode:
                   type: string
                   description: Unique account code to pay from
+    ManualLedgerCheck:
```

Comment:
This has brought up questions over the content of the payloads being sent and how they can be reduced to aid in maintainability -- payload properties which need assessed recorded before

Reply:
https://github.com/DEFRA/ffc-pay-processing/pull/55#discussion_r828300221

Payload constructed as advised by John